safe-global/safe-wallet-web
# feat: display legal disclaimer when swapping for the first time

## What it solves

Resolves: https://www.notion.so/safe-global/ToS-7965c981bc1c45839454b8499ec296b4?pvs=4

## How this PR fixes it
- Display the legal disclaimer if user has not confirmed the disclaimer already.
- Still show the disclaimer if the user has accepted through the safe app.

## How to test it
- Go to the swaps feature in a private window.
- It should display the disclaimer.
- After accepting the disclaimer, it should not be shown upon refresh or revisiting the swaps feature.

## Screenshots
<img width="952" alt="image" src="https://github.com/safe-global/safe-wallet-web/assets/17801424/b90329d2-1f14-4d22-a5bf-aedd22ff1384">


## Checklist
